src/SharpLearning.Optimization.Test/GlobalizedBoundedNelderMeadOptimizerTest.cs

```diff
@@ -8,15 +8,19 @@ namespace SharpLearning.Optimization.Test
     public class GlobalizedBoundedNelderMeadOptimizerTest
     {
         [TestMethod]
-        public void GlobalizedBoundedNelderMeadOptimizer_OptimizeBest()
+        [DataRow(1)]
```

Comment:
@jameschch Nice with the `DataRow` parameters, I haven't used those before, so I learned something in this PR. Thanks!